Goal: kolab/issue1382 and kolab/issue2499

Prepare kolabd for kolab/issue1382 (resmgr accepts concurrent overlapping
invitations) and kolab/issue2499 (Notification messages by the resource
manager sould be localized)

Author: Gunnar Wrobel  <p@rdus.de>

Upstream status: Already commit on TRUNK

Index: b/templates/resmgr.conf.template.in
===================================================================
--- a/templates/resmgr.conf.template.in
+++ b/templates/resmgr.conf.template.in
@@ -184,6 +184,17 @@
 //$conf['kolab']['filter']['locale_path'] = '@resmgr_localedir@';
 //$conf['kolab']['filter']['locale'] = 'de_DE';
 
+/* Uncomment if the resource manager should generate translated messages. */
+//$conf['kolab']['filter']['locale_path'] = '@resmgr_localedir@';
+//$conf['kolab']['filter']['locale'] = 'de_DE';
+
+/* Use locking for the resource manager? */
+$conf['kolab']['filter']['simple_locks'] = true;
+/* Timeout for the locks */
+$conf['kolab']['filter']['simple_locks_timeout'] = 60;
+/* Directory for storing the locks */
+$conf['kolab']['filter']['simple_locks_dir'] = '@resmgr_locksdir@';
+
 /* PHP error logging */
 $conf['kolab']['filter']['error_log'] = '@resmgr_logdir@/fatal.log';
 
Index: b/dist_conf/common
===================================================================
--- a/dist_conf/common
+++ b/dist_conf/common
@@ -120,6 +120,7 @@
 	-e 's=[@]resmgr_conffile_usr[@]=$(resmgr_conffile_usr)=g' \
 	-e 's=[@]resmgr_tmpdir[@]=$(resmgr_tmpdir)=g' \
 	-e 's=[@]resmgr_grp[@]=$(resmgr_grp)=g' \
+	-e 's=[@]resmgr_locksdir[@]=$(resmgr_locksdir)=g' \
 	-e 's=[@]resmgr_localedir[@]=$(resmgr_localedir)=g' \
 	-e 's=[@]resmgr_logdir[@]=$(resmgr_logdir)=g' \
 	-e 's=[@]resmgr_scriptsdir[@]=$(resmgr_scriptsdir)=g' \
